### PR TITLE
Add October 2022 newsletter to website

### LINKED ISSUE #644: Add October 2022 newsletter to the website
## Issue Context

Every month, we try to get the newsletter up on the site within a week of emailing it. Currently, we're moving them over "by hand."

## Steps to update

In the code base, navigate to `app > routes > newsletter > issues` and create a new file `2022-10.jsx`.
You can look at the existing newsletters ( `app > routes > newsletter > issues`) as a template.

Make sure to add it to the index by following the steps in [Newsletters section in our README](https://github.com/Virtual-Coffee/virtualcoffee.io#newsletters) and update the content accordingly based on our email newsletter.

If you have questions, please let us know. We're up for pairing if anyone wants to walk through this!

### PR DESCRIPTION
**Pull request by adiati98**
## Linked Issue

Closes #644 

## Description

- Add the October 2022 newsletter.
- Update `newsletter.js` to add the issue to the index.

## Code of Conduct

> By submitting this pull request, you agree to follow our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/)

### CODE REVIEW

File: app/routes/__frontend/newsletter/issues/2022-10.jsx [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+			</blockquote>
+
+			<hr />
+
+			<h2 className="my-5">📆 What's happening at Virtual Coffee</h2>
+
+			<h3 className="font-italic">September Recap</h3>
+			<p>
+				Monthly challenge -&gt;{' '}
+				<a href="https://virtualcoffee.io/monthlychallenges/sept-2022">
```
Comment (danieltott) [inline app/routes/__frontend/newsletter/issues/2022-10.jsx:53]: Can we try to use `Link` for internal links?

```suggestion
				<Link to="/monthlychallenges/sept-2022">
```

**Review by danieltott — CHANGES_REQUESTED** [review]
Hey @adiati98 -

I know we haven't been doing this a lot previously with the newsletters, but I'd like to try to use `Link` for internal links going forward (for all content).

We'll need to import it at the top of the file:

```js
import { Link } from '@remix-run/react';
```

And then we can replace any `<a href=...` with `<Link to=...` for any links that are on our site (any links starting with `https://virtualcoffee.io/`

Let me know if that doesn't make sense

**Comment by adiati98** [comment]
@danieltott let me do the revisions and I'll get back to you whenever I have questions 😊

Do you want the revision only in October's newsletter, or within all newsletters?

**Comment by danieltott** [comment]
Just this one is fine for now 👍🏼

**Comment by adiati98** [comment]
@danieltott I've replaced the `a href` with `link to` and pushed the changes.
Please let me know if I need to revise anything at all 😄

**Comment by adiati98** [comment]
@danieltott I'm already away from my computer and just realized that I haven't update this branch with `main` 🤦🏻‍♀️

Sorry! If I need to do that, I'll do it tomorrow morning 😅

Thanks, Dan!

**Review by danieltott — APPROVED** [review]
approved

**Comment by adiati98** [comment]
@danieltott 
woop woop GIF

**Pull request merged**